pkg/scheduler/plugins/overcommit/overcommit.go

```diff
@@ -37,23 +39,39 @@ const (
 	defaultOverCommitFactor = 1.2
 )
 
+const (
+	// overCommitFactorPrefix is the prefix of resource overCommit factor
+	overCommitFactorPrefix = "overcommit-factor."
+)
+
+// overcommitFactors defines the resource overCommit factors
+type overcommitFactors struct {
+	// factorMaps defines the resource overCommit factors
+	// key: resource, example: "cpu", "memory", "ephemeral-storage", "nvidia.com/gpu"
+	// value: overCommit factors
+	factorMaps map[string]float64
```

Comment:
1. need a defaultFactor to store the configured value in overcommit-factor. If it is set by user, updating to it, otherwise default is `defaultOverCommitFactor`.
2. please add ut to cover your parse function.

Reply:
Thanks, that's a good idea.